scripts/check_compliance.py

```diff
@@ -172,7 +172,7 @@ def run(self):
                                 stdout=subprocess.PIPE)
         try:
             subprocess.check_output((checkpatch, '--mailback', '--no-tree', '-'),
-                                    stdin=diff.stdout,
+                                    stdin=diff.stdout, cwd=git_top,
```

Comment:
Bit odd to have `cwd` sandwiches between `stdin` and `stderr`.

Reply:
(Pointless) effort to minimize the diff size; will re-order.